Source/Lib/Codec/EbSystemResourceManager.c

```diff
@@ -427,12 +427,12 @@ static EB_ERRORTYPE EBObjectWrapperCtor(EbObjectWrapper_t* wrapper,
     EB_ERRORTYPE ret;
 
     wrapper->dctor = EBObjectWrapperDctor;
-    ret = objectCreator(&wrapper->objectPtr, objectInitDataPtr);
-    if (ret != EB_ErrorNone)
-        return ret;
     wrapper->releaseEnable = EB_TRUE;
```

Comment:
If objectCreator return error, objectDestroyer is still NULL, crash happens when `EBObjectWrapperDctor` is called, because it goes to:
```
 else {
       //hack....
```